### PR TITLE
add command to export dg

### PR DESCRIPTION
**Pull request by ansyral**

### CODE REVIEW

**Comment by superyyrrzz** [comment]
:shipit:

File: src/docfx/SubCommands/DependencyCommand.cs [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+            var buildInfo = BuildInfo.Load(intermediateFolder);
+            if (buildInfo == null)
+            {
+                Logger.LogWarning($"Cache files in the folder {intermediateFolder} are corrupted!");
+                return;
+            }
+            var dg = buildInfo.Versions.FirstOrDefault(v => v.VersionName == versionName)?.Dependency;
+            if (dg == null)
+            {
+                Logger.LogWarning($"Cache files for version {versionName} is not found!");
```
Comment (herohua) [inline src/docfx/SubCommands/DependencyCommand.cs]: should be error

File: src/docfx/SubCommands/DependencyCommandCreator.cs [hunk 2]
```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DocAsCode.SubCommands
+{
+    using Microsoft.DocAsCode;
+    using Microsoft.DocAsCode.Plugins;
+
+    [CommandOption("dependency", "Export dependency file")]
```
Comment (herohua) [inline src/docfx/SubCommands/DependencyCommandCreator.cs]: can we define a subcommand as `export`? I think there might be other cases we want to add into the `dependency` command

File: src/docfx/SubCommands/DependencyCommand.cs [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    using System.Linq;
+
+    using Microsoft.DocAsCode;
+    using Microsoft.DocAsCode.Build.Engine.Incrementals;
+    using Microsoft.DocAsCode.Build.Engine.Incrementals.Outputs;
+    using Microsoft.DocAsCode.Common;
+    using Microsoft.DocAsCode.Exceptions;
+    using Microsoft.DocAsCode.Plugins;
+
+    internal sealed class DependencyCommand : ISubCommand
```
Comment (hellosnow) [inline src/docfx/SubCommands/DependencyCommand.cs]: >DependencyCommand [](start = 26, length = 17)

add some tests? refer to ArgsParserTest and BuildCommandTest

**Pull request merged**